File: Project_DoAnNhanhSOL/app/src/main/java/com/example/project_doannhanhsol/DTO/Product.java
package com.example.project_doannhanhsol.DTO;

public class Product {
    private int Id_product,Id_danhmuc,sales,views;
    private String NameProduct,Content,Imagelinks,JoinDate;

    public Product(int id_product, int id_danhmuc, int sales, int views, String nameProduct, String content, String imagelinks, String joinDate) {
        Id_product = id_product;
        Id_danhmuc = id_danhmuc;
        this.sales = sales;
        this.views = views;
        NameProduct = nameProduct;
        Content = content;
        Imagelinks = imagelinks;
        JoinDate = joinDate;
    }

    public int getId_product() {
        return Id_product;
    }

    public int getId_danhmuc() {
        return Id_danhmuc;
    }

    public int getSales() {
        return sales;
    }

    public int getViews() {
        return views;
    }

    public String getNameProduct() {
        return NameProduct;
    }

    public String getContent() {
        return Content;
    }

    public String getImagelinks() {
        return Imagelinks;
    }

    public String getJoinDate() {
        return JoinDate;
    }
}

File: WebApplication3/WebApplication3/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication3.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Product")]
    public partial class Product
    {
        [Key]
        public int Id_product { get; set; }

        public int? Id_danhmuc { get; set; }

        public string NameProduct { get; set; }

        public string Content { get; set; }

        public int? sales { get; set; }

        public int? views { get; set; }

        public string Imagelinks { get; set; }

        public string JoinDate { get; set; }
    }
}